packages/platform-core/src/parameter_path_conversions.ts

```diff
@@ -34,7 +35,17 @@ export class ParameterPathConversions {
  * Get a branch-specific parameter prefix.
  */
 const getBranchParameterPrefix = (parts: BackendIdentifier): string => {
```

Comment:
this was already here but if this function covers both sandboxes and branches then it's name is kind of wrong.

Reply:
Updating to `getBackendParameterPrefix`